janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/AbstractIndexStatusWatcher.java

```diff
@@ -85,5 +91,13 @@ public S pollInterval(long poll, TemporalUnit pollUnit) {
         this.poll = Duration.of(poll, pollUnit);
         return self();
     }
+    
+    boolean hasStatuses(SchemaStatus status) {
```

Comment:
This method is equivalent to `statuses.contains(status)`. Can you remove this method and update usages to with this instead?